README.md

```diff
@@ -1881,6 +1881,7 @@ Update Time, five active automations, webhooks.
   * [snappify](https://snappify.com) - Enables developers to create stunning visuals. From beautiful code snippets to fully fletched technical presentations. The free plan includes up to 3 snaps at once with unlimited downloads and 5 AI-powered code explanations per month.
   * [Sunrise and Sunset](https://sunrisesunset.io/api/) - Get sunrise and sunset times for a given longitude and latitude.
   * [superfeedr.com](https://superfeedr.com/) — Real-time PubSubHubbub compliant feeds, export, analytics. Free with less customization
+  * [Survicate](https://survicate.com/) — Collect feedback from all sources. Automatically analyze, categorize and consolidate it with Feedback Hub and AI Insights Assistant to extract intelligence and make quicker decisions. Connect with dozens of third-party tools using native integrations.
```

Comment:
thanks, please add a short indication of whats free like in other entries

Reply:
thanks for the quick response, I've added the changes